.jekyll-cache/Jekyll/Cache/Jekyll--Converters--Markdown/c7/71564f874a8bb83404d37c922fd6727a44d1cbcbee928c1d9c82a3c2d52fb6

```diff
@@ -0,0 +1,32 @@
+I"<table>
+  <thead>
+    <tr>
+      <th>Test Instruction</th>
+      <th>Instruction Detail</th>
+      <th>Expected Test Case Result</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><strong>IC-1</strong></td>
+      <td>Identify any <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> elements.</td>
+      <td>One <code class="language-plaintext highlighter-rouge">&lt;img&gt;</code> exists in the test data code sample.</td>
+    </tr>
+    <tr>
+      <td><strong>IC-1</strong></td>
+      <td>Determine whether the image is meaningful. Review the page content and image context to determine whether the image conveys meaningful information.</td>
+      <td>The image is meaningful, based on the text directly below the image.</td>
+    </tr>
+    <tr>
+      <td><strong>6.1</strong></td>
+      <td>Calculate the accessible name and accessible description output for the image according to <a href="https://www.w3.org/TR/html-aam-1.0/#img-element">HTML Accessibility API Mappings 1.0 for img</a>.</td>
```

Comment:
Should be a copy and paste from 6.1-1: "Check that the text alternative (combination of the accessible name and accessible description) is not empty."

There are 5 total test instructions in 6.1. Are we only including up to the instruction(s) that the test file fails?

Reply:
Yes, my preference would be to only include the bare minimum number of steps necessary to validate alignment for a particular test case / code sample with the baseline. Including other steps when we already know the result just adds more space to the alignment validation report, and it takes more time for people to read and complete. Just seems unnecessary.

Reply:
I think editing the Baseline in the alignment shouldn't be done. When others start contributing test cases, it will be harder to manage if it's not a copy & paste. I'm good with only listing the test instructions needed for the test case.

Reply:
Fixed the text in the 6.1 result. Straight copy and paste from the result now.